packages/rx_bloc_cli/mason_templates/bricks/lib_internationalisation/__brick__/lib/lib_internationalisation/services/language_service_example.dart

```diff
@@ -0,0 +1,25 @@
+{{> licence.dart }}
+
+import 'package:widget_toolkit/language_picker.dart';
+
+import '../data_sources/language_local_data_source.dart';
+
+/// Example implementation of the LanguageService
+class LanguageServiceExample extends LanguageService {
```

Comment:
What the example stands for ?

Reply:
This was an example for a custom language service implementation, so I renamed it to `CustomLanguageService`

Reply:
Please use the prefix App -> `AppLanguageService`